lib/build.js

```diff
@@ -6,15 +6,27 @@ const purge = require('./purge')
 // Conditionally require node-sass or dart-sass
 const pathToNodeSass = path.normalize(path.resolve(process.cwd(), 'node_modules', 'node-sass'))
 const pathToDartSass = path.normalize(path.resolve(process.cwd(), 'node_modules', 'sass'))
-let sass = null
 
-try {
-  sass = require(pathToNodeSass)
-} catch (err) {
+function defineSass (sassType) {
+  if (sassType === true) {
+    try {
+      return require(pathToNodeSass)
+    } catch {
+      try {
+        return require(pathToDartSass)
+      } catch {
+        return null
+      }
+    }
+  }
   try {
-    sass = require(pathToDartSass)
-  } catch (e) {
-    sass = null
+    if (sassType === 'node-sass') {
+      return require(pathToNodeSass)
+    } else if (sassType === 'sass' || sassType === 'dart–sass') {
```

Comment:
Some minor adjustments required, but no worries, I'll take care of them. Did you know you used a non-standard hyphen character at `'dart-sass'`? That wasn't on purpose, was it?